deploy/olm-catalog/kubevirt-hyperconverged/0.0.2/kubevirt-hyperconverged-operator.v0.0.2.clusterserviceversion.yaml

```diff
@@ -55,6 +55,13 @@ metadata:
             "name": "template-validator"
           }
         },
+        {
+          "apiVersion": "kubevirtmetricsaggregations.kubevirt.io/v1",
```

Comment:
weird. this shouldn't be added by you.

Reply:
This happened when I ran the build-manifest. Guessing maybe somebody did an update and didn't?